README.md

```diff
@@ -81,10 +113,9 @@ changes from feature branches into your `main` branch.
   - [CSS Box Model](http://www.w3schools.com/css/css_boxmodel.asp)
   - [CSS Float](http://www.w3schools.com/css/css_float.asp)- How to position elements next to each other
   - [CSS Display and Visibility](http://www.w3schools.com/css/css_display_visibility.asp)
-  - [CSS3 Features](http://www.w3schools.com/css/css3_intro.asp)- Relatively recent stuff, cool features not yet supported by all browsers.
 - [CSS Zen Garden](http://www.csszengarden.com/)- Some fancy examples of CSS, all of which use the same HTML, but look wildly different
-- [Daily CSS Fun](https://placenamehere.com/neuralustmirror/200202/)- Examples of several CSS pages altering the same HTML
-- [FontJoy](http://fontjoy.com) – a cool tool for helping find combinations of fonts that work well together
+- [FontJoy](https://fontjoy.com/) - a nice tool for helping find combinations of fonts that work well together. (Google Fonts provides a similar feature as well.)
 - [Colormind](http://colormind.io) – a cool tool for helping find color combinations that work well together
 - [CSS Flexbox](https://www.w3schools.com/css/css3_flexbox.asp) - Add responsive design to automatically scale elements with page size
```

Comment:
I've found myself using this resource frequently for flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Also this playground tool has been useful: https://demos.scotch.io/visual-guide-to-css3-flexbox-flexbox-playground/demos/ (there is something similar but specific to Angular we can include in future labs)